registration/registration-client/src/main/java/io/mosip/registration/controller/auth/AuthenticationController.java

```diff
@@ -346,7 +346,7 @@ public void validateFingerprint() {
 			executeFPValidationTask(fpUserId.getText(), operatorAuthenticationPane);
 		}
 
-		authCounter.setText(++fingerPrintAuthCount + "");
+//		authCounter.setText(++fingerPrintAuthCount + "");

```

Comment:
Why is the counter removed?

Reply:
https://mosip.atlassian.net/browse/MOSIP-23225 - Defect was raised for showing counter as it is not necessary. 
Rounak asked to remove the counter.